tasks/broad/GermlineVariantDiscovery.wdl

```diff
@@ -209,6 +219,43 @@ task HardFilterVcf {
   }
 }
 
+# This hard filtering matches DRAGEN 3.4.12. For later DRAGEN versions, this needs to be updated.
+task DragenHardFilterVcf {
+  input {
+    File input_vcf
+    File input_vcf_index
+    Boolean make_gvcf
+    String vcf_basename
+    Int preemptible_tries
+    String gatk_docker = "us.gcr.io/broad-gatk/gatk:4.2.2.0"
+  }
+
+  Int disk_size = ceil(2 * size(input_vcf, "GiB")) + 20
+
+  String output_suffix = if make_gvcf then ".g.vcf.gz" else ".vcf.gz"
+  String output_vcf_name = vcf_basename + ".hard-filtered" + output_suffix
+
+  command {
+     gatk --java-options "-Xms3000m" \
+      VariantFiltration \
+      -V ~{input_vcf} \
+      --filter-expression "QUAL < 10.4139" \
+      --filter-name "DRAGENHardQUAL" \
+      -O ~{output_vcf_name}
+  }
+  output {
+    File output_vcf = "~{output_vcf_name}"
+    File output_vcf_index = "~{output_vcf_name}.tbi"
+  }
+  runtime {
+    docker: gatk_docker
+    preemptible: preemptible_tries
+    memory: "3 GiB"
```

Comment:
Another place to consider including memory in the inputs or scaling based on input size. If this is not a very memory hungry task then hard coding a small value may be fine

Reply:
I used the same configuration as the HardFilterVcf task right above this task. Let me know if you want me to change it.